src/components/SearchableSelect/SearchableSelect.jsx

```diff
@@ -405,88 +416,109 @@ const SearchableSelect = createClass({
 		const placeholderProps = _.first(
 			_.map(findTypes(this.props, SearchableSelect.Placeholder), 'props')
 		);
+		const errorChildProps = _.first(
+			_.map(findTypes(props, Validation.Error), 'props')
+		);
+		
+
 		const placeholder = _.get(placeholderProps, 'children', 'Select');
 		const isItemSelected = _.isNumber(selectedIndex);
 
 		return (
-			<DropMenu
-				{...dropMenuProps}
+			<div Error={errorChildProps}
 				className={cx('&', className)}
-				optionContainerStyle={_.assign(
-					{},
-					optionContainerStyle,
-					!_.isNil(maxMenuHeight) ? { maxHeight: maxMenuHeight } : null
-				)}
-				isDisabled={isDisabled}
-				onSelect={onSelect}
-				selectedIndices={isItemSelected ? [selectedIndex] : []}
 				style={style}
 			>
-				<DropMenu.Control>
-					<div
-						tabIndex={0}
-						className={cx('&-Control', {
-							'&-Control-is-highlighted':
-								(!isDisabled && isItemSelected && isSelectionHighlighted) ||
-								(isExpanded && isSelectionHighlighted),
-							'&-Control-is-selected':
-								!isDisabled && isItemSelected && isSelectionHighlighted,
-							'&-Control-is-expanded': isExpanded,
-							'&-Control-is-invisible': isInvisible,
-							'&-Control-is-disabled': isDisabled,
-						})}
-					>
-						<span
-							{...(!isItemSelected ? placeholderProps : null)}
-							className={cx(
-								'&-Control-content',
-								!isItemSelected ? _.get(placeholderProps, 'className') : null
-							)}
+				<DropMenu
```

Comment:
It's a little tough to see the diff because of formatting stuff. Maybe run prettier on this and then it'll be more clear?